Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ghost:6.5.3-alpine
+
+# Add the Object Store storage adapter
+ADD https://github.com/CodeForAfrica/ghost-object-store-storage-adapter.git content/adapters/storage/object-store
+
+# Install dependencies for the storage adapter
+WORKDIR /var/lib/ghost/content/adapters/storage/object-store
```

Comment:
I'm not entirely sure how this installation happens, but I think we need to ensure correct ownership of the adapter. 
I see ownership within `/var/lib/ghost/content` should be `node:node` ,but with the installation above, we get this

```
/var/lib/ghost/content/adapters # ls -la
total 12
drwxr-xr-x    3 node     root          4096 Nov  4 07:05 .
drwxrwxrwt   12 node     node          4096 Nov  4 07:06 ..
drwxr-xr-x    3 node     root          4096 Nov  4 07:05 storage
/var/lib/ghost/content/adapters # ls -la
total 12
drwxr-xr-x    3 node     root          4096 Nov  4 07:05 .
drwxrwxrwt   12 node     node          4096 Nov  4 07:06 ..
drwxr-xr-x    3 node     root          4096 Nov  4 07:05 storage`

Reply:
Good catch. I've fixed this with a `chown`.